kaiterra_laser_egg_exporter.go

```diff
@@ -148,6 +148,17 @@ func (kc kaiterraCollector) Collect(ch chan<- prometheus.Metric) {
 		prometheus.GaugeValue,
 		float64(t.Unix()),
 	)
+	ch <- prometheus.MustNewConstMetric(
```

Comment:
So I just checked, and my cheap laser egg doesn't have this AQI.Data.Tvoc in its json, it only has:

```
{"id":"elided","info.aqi":{"ts":"2022-07-10T01:15:32Z","data":{"humidity":64.51,"pm10":0,"pm25":0,"temp":18.74}}}
```

I wonder what we should do in situations where there's no Tvoc data. I'm guessing we should not emit the metric -- that is, wrap this `ch <- ` statement in a `if Tvoc != 0 {` if-statement? This would be my best guess for what to do here. No point emitting (and paying for the storage of) a metric that is inapplicable, right?

But perhaps that's not the 'proper' way in prometheus, perhaps the 'proper' way is to emit a metric with 0? I don't think so, but I'm not really a prometheus expert.